exporter/kafkaexporter/otlp_marshaller_test.go

```diff
@@ -21,24 +21,39 @@ import (
 	"github.com/stretchr/testify/require"
 
 	"go.opentelemetry.io/collector/consumer/pdata"
+	otlpmetric "go.opentelemetry.io/collector/internal/data/opentelemetry-proto-gen/collector/metrics/v1"
 	otlptrace "go.opentelemetry.io/collector/internal/data/opentelemetry-proto-gen/collector/trace/v1"
+	"go.opentelemetry.io/collector/internal/data/testdata"
 )
 
-func TestOTLPMarshaller(t *testing.T) {
-	td := pdata.NewTraces()
-	td.ResourceSpans().Resize(1)
-	td.ResourceSpans().At(0).Resource().InitEmpty()
-	td.ResourceSpans().At(0).Resource().Attributes().InsertString("foo", "bar")
+func TestOTLPTracesPbMarshaller(t *testing.T) {
+	td := testdata.GenerateTraceDataTwoSpansSameResource()
 	request := &otlptrace.ExportTraceServiceRequest{
 		ResourceSpans: pdata.TracesToOtlp(td),
 	}
 	expected, err := request.Marshal()
 	require.NoError(t, err)
 	require.NotNil(t, expected)
 
-	m := otlpProtoMarshaller{}
+	m := otlpTracesPbMarshaller{}
 	assert.Equal(t, "otlp_proto", m.Encoding())
 	messages, err := m.Marshal(td)
 	require.NoError(t, err)
 	assert.Equal(t, []Message{{Value: expected}}, messages)
 }
+
+func TestOTLPMetricsPbJSONMarshaller(t *testing.T) {
```

Comment:
Why does this have "JSON" in the name? Does it do anything related to JSON? This is binary Protobuf marshaler test, right?

Reply:
Yes it is only a Proto test, this is a typo, renaming it now.